tests/unit/filings.json

```diff
@@ -519,6 +521,33 @@
       "status": "COMPLETED",
       "submittedDate": "Sun, 01 Oct 2017 00:02:26 GMT",
       "submitter": "Registry Staff"
+    },
+    {
+      "availableOnPaperOnly": false,
+      "businessIdentifier": "CP0000840",
```

Comment:
I don't know if it makes any difference in the code, but is this a valid entity type for administrative dissolution?

Actually this is the business ID; the entity type is set elsewhere. Maybe this test suite needs to be laid out different for CP vs BC vs FM... can be future work (as needed) if the current tests aren't broken by this.